package-lock.json

```diff
@@ -12726,7 +12722,8 @@
     "util-deprecate": {
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/util-deprecate/-/util-deprecate-1.0.2.tgz",
-      "integrity": "sha1-RQ1Nyfpw3nMnYvvS1KKJgUGaDM8="
+      "integrity": "sha1-RQ1Nyfpw3nMnYvvS1KKJgUGaDM8=",
+      "dev": true
```

Comment:
what is dev: true in the lock file ?

Reply:
As I understand the `util-deprecate` was used by `bs58check` (not a dev dependency), and since it was removed, then I assume there is some other package that still depends on `util-deprecate`.

Reply:
found it in the docs :)